website/src/main.js

```diff
@@ -0,0 +1,12 @@
+import App from './App.svelte'
+
+import 'bootstrap/dist/css/bootstrap.min.css'
+
+const app = new App({
+  target: document.body,
+  props: {
+    name: 'world'
```

Comment:
Should it be around?

Reply:
Nope this props come with the initial boilerplate I removed it 👍